softioc/asyncio_dispatcher.py

```diff
@@ -47,6 +48,25 @@ def close(self):
             self.__atexit = None
 
         self.__shutdown()
+    
+    def wait_for_quit(self):
+
+        async_stop_event = asyncio.Event()
+        stop_event = threading.Event()
```

Comment:
It seems odd to me to have two signals, where one just chains into the other. However I see in your commits that these were added in response to a bug. Could you please elaborate on the problem of having just one Event?